update developer cocs (#3393)
dfinity/portal: docs/developer-docs/multi-chain/overview.mdx

## How to build applications using Chain Fusion

### Example code

> To showcase how powerful Chain Fusion is, here is a simple example that shows three chains interacting in one smart contract: a single **ICP** smart contract that can custody **Bitcoin** and programmatically trigger sending it based on events observed on a **Ethereum** DeFi smart contract.

Reviewer: ```suggestion
To showcase how powerful Chain Fusion is, here is a simple example that shows three chains interacting in one smart contract: a single **ICP** smart contract that can have custody of **bitcoin** and programmatically trigger sending it based on events observed on an **Ethereum** DeFi smart contract.
```

> This code snippet is written in both Rust and the Motoko programming language but can also be written in TypeScript, Python, and other languages.

Reviewer: ```suggestion
This code snippet is written in both Rust and Motoko, but a similar example can also be written in TypeScript, Python, and other languages.
```
